### PR TITLE
Add Scaleway Icon

### PR DESCRIPTION
**Pull request by kindermoumoute**
### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I'm part of the dev tools team [@scaleway](https://github.com/scaleway). The SVG was manually exported for this repo.

### CODE REVIEW

**Comment by davidklebanoff** [comment]
Hi @kindermoumoute,

It looks like Travis caught that the icon has multiple `<path>` elements. Can you compress it down to a single path?

**Comment by kindermoumoute** [comment]
@davidklebanoff yes, I'm still trying to figure out how to do that but I won't update the PR until Thursday.

File: icons/scaleway.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Scaleway</title><path d="M11.1 18.5h4c.9-.1 1.5-.8 1.6-1.7v-5.7c0-.8-.6-1.4-1.4-1.4-.8 0-1.4.6-1.4 1.4v3.6c0 .6-.5 1-1 1h-1.8c-.4 0-.7.1-1 .4-.5.5-.6 1.4 0 1.9.3.4.7.5 1 .5zm1.8-13H9c-.9.1-1.6.8-1.6 1.7v5.7c0 .8.6 1.4 1.4 1.4.8 0 1.4-.6 1.4-1.4V9.3c0-.6.4-1 1-1H13c.4 0 .7-.1 1-.4.5-.5.5-1.4 0-1.9-.4-.4-.8-.5-1.1-.5z"/><path d="M13.6 2.7c3.2 0 5.8 2.6 5.8 5.8v11.2c-.1.8-.8 1.4-1.5 1.5h-7.5c-3.2 0-5.8-2.6-5.8-5.8V4.5c0-1 .8-1.8 1.8-1.8h7.2m0-2.7H6.4C3.9 0 1.9 2 1.9 4.5v10.9c0 4.7 3.8 8.6 8.6 8.6H18c2.2-.2 3.9-1.9 4.1-4V8.6c.1-4.7-3.7-8.5-8.5-8.6z"/></svg>
```
Comment (hylickipiotr) [inline icons/scaleway.svg]: @kindermoumoute I think `<title>` should contain `Scaleway icon`.

**Comment by davidklebanoff** [comment]
@kindermoumoute - Do you need assistance with this icon? Would you like someone else to complete the last steps? It looks like you've completed 90% of the work!

**Comment by hylickipiotr** [comment]
@davidklebanoff I will finish this PR.

**Comment by kindermoumoute** [comment]
Thanks @hylickipiotr, I didn't find a way to merge these 2 paths into one. 🤔

**Comment by ericcornelissen** [comment]
> Thanks @hylickipiotr, I didn't find a way to merge these 2 paths into one. 🤔

What editor are you using to edit the SVG, if you tell me I can help you find how to do that? Generally speaking it is usually called either **merge paths** or **compound paths**

**Comment by kindermoumoute** [comment]
Thanks for the help, looks like this one is passing tests. 🎉

**Comment by ericcornelissen** [comment]
Thanks for the addition @kindermoumoute, it is much appreciated 🎉 I hope you learned something along the way as well 😄

**Pull request merged**